Makefile

```diff
@@ -16,6 +16,8 @@ VERSION := 0.9
 SBIN_FILES := src/stashcache
 INSTALL_SBIN_DIR := usr/sbin
 XROOTD_CONFIG := configs/Authfile-auth configs/Authfile-noauth configs/stashcache-robots.txt configs/xrootd-stashcache-cache-server.cfg configs/xrootd-stashcache-origin-server.cfg
+XROOTD_CONFIGD := configs/config.d/40-osg-http.cfg configs/config.d/40-osg-monitoring.cfg configs/config.d/40-osg-xcache.cfg configs/config.d/50-stashcache-authz.cfg configs/config.d/50-stashcache-logging.cfg configs/config.d/10-osg-site-local.cfg
```

Comment:
Can we put each file onto separate lines? It'll make diffs clearer in the future

Reply:
Done.  Also nuked a few nearby overly-long lines.